src/components/Tool/Appeal/List/AppealsListFilterPanel/AddContactToAppealModal/AddContactToAppealModal.tsx

```diff
@@ -0,0 +1,122 @@
+import React, { ReactElement } from 'react';
+import { DialogActions, DialogContent, Grid } from '@mui/material';
+import { Formik } from 'formik';
+import { useSnackbar } from 'notistack';
+import { useTranslation } from 'react-i18next';
+import * as yup from 'yup';
+import { ContactsAutocomplete } from 'src/components/Task/Modal/Form/Inputs/ContactsAutocomplete/ContactsAutocomplete';
+import {
+  CancelButton,
+  SubmitButton,
+} from 'src/components/common/Modal/ActionButtons/ActionButtons';
+import Modal from 'src/components/common/Modal/Modal';
+import {
+  AppealsContext,
+  AppealsType,
+} from '../../../AppealsContext/AppealsContext';
+import { useAssignContactsToAppealMutation } from './AddContactToAppeal.generated';
+import { useAppealQuery } from './appealInfo.generated';
+
+interface AddContactToAppealModalProps {
+  handleClose: () => void;
+}
+
+export type AddContactFormikSchema = {
+  contactIds: string[];
+};
+
+const AddContactSchema: yup.SchemaOf<AddContactFormikSchema> = yup.object({
+  contactIds: yup.array().of(yup.string().required()).default([]),
+});
+
+export const AddContactToAppealModal: React.FC<
+  AddContactToAppealModalProps
+> = ({ handleClose }) => {
+  const { t } = useTranslation();
+  const { enqueueSnackbar } = useSnackbar();
+  const [assignContactsToAppeal] = useAssignContactsToAppealMutation();
+  const { accountListId, appealId, contactsQueryResult } = React.useContext(
+    AppealsContext,
+  ) as AppealsType;
+
+  const { data } = useAppealQuery({
+    variables: {
+      accountListId: accountListId ?? '',
+      appealId: appealId ?? '',
+    },
+  });
+
+  const existingContactIds = data?.appeal?.contactIds ?? [];
+
+  const onSubmit = async (attributes: AddContactFormikSchema) => {
```

Comment:
Could this also add the contact to the 'Asked' category?

Reply:
Currently, there isn't a filter to set that. I think the issue is that I'm not using the correct filter to grab the asked. However, there isn't a filter for it, so I'm talking to Andrew about that one too.

Reply:
I wonder if filters aren't the way it is being organized in the angular.

Reply:
Yeah, I think it's the way graphQL handles the filtering in the backend.